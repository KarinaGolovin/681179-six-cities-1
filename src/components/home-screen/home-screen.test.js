import React from 'react';
import HomeScreen from './home-screen.jsx';
import ShallowRenderer from 'react-test-renderer/shallow';

const renderer = new ShallowRenderer();

it(`HomeScreen correctly renders`, () => {
  const component = renderer.render(
      <HomeScreen selectedCityName={`Test`}/>
  );

  expect(component).toMatchSnapshot();
});

