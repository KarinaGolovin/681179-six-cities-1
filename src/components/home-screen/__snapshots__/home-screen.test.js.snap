// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen correctly renders 1`] = `
<Memo(ConnectFunction)>
  <Memo(ConnectFunction) />
</Memo(ConnectFunction)>
`;
